mb-reledit-set_relation_attrs.user.js

```diff
@@ -41,7 +48,7 @@ function setAttributes(relationType, attrId, toggle) {
 (function displayToolbar() {
     relEditor.container(document.querySelector('div.tabs'))
              .insertAdjacentHTML('beforeend', `
-        <h3><span id="relattrs_script_toggle">▶ Relation attributes</span></h3>
+        <h3><span id="relattrs_script_toggle" style="cursor:pointer">▶ Relation attributes</span></h3>
```

Comment:
@loujine, do you prefer the following, to pass this more than 100 characters check?

```html
        <h3>
            <span id="relattrs_script_toggle" style="cursor:pointer">▶ Relation attributes</span>
        </h3>
```

Reply:
Yes, that looks good. I also usually use a 2-spaces indentation on HTML in JS files (just fixed it on that file with your other PR)